app/models/progress_report.rb

```diff
@@ -5,15 +5,30 @@ class ProgressReport < VersionedRecord
   has_many :recommendations, through: :measures
   has_many :categories, through: :recommendations
   has_one :manager, through: :indicator
+  delegate :email, to: :manager, prefix: true, allow_nil: true
+  delegate :name, to: :manager, prefix: true, allow_nil: true
```

Comment:
is this the correct manager @tmfrnz ?

Reply:
also see https://github.com/impactoss/impactoss-server/pull/386

Reply:
@parndt following up on my previous comment (https://github.com/impactoss/impactoss-server/pull/399#discussion_r1593873436), the relationships between category managers and indicators can also include parent/child categories

* users (category manager) > categories (parent category) > categories (child category) > recommendation_categories > recommendation > recommendation_indicators > indicators
* users (category manager) > categories (parent category) > categories > recommendation_categories > recommendation > recommendation_measures > measures > measure_indicators > indicators > indicators